tornado/platform/asyncio.py

```diff
@@ -207,15 +193,7 @@ def _handle_events(self, fd: int, events: int) -> None:
         handler_func(fileobj, events)
 
     def start(self) -> None:
-        try:
-            old_loop = _get_event_loop()
-        except (RuntimeError, AssertionError):
-            old_loop = None  # type: ignore
-        try:
-            asyncio.set_event_loop(self.asyncio_loop)
```

Comment:
because `asyncio.get_event_loop()` calls `asyncio.get_running_loop()` this `set_event_loop()` is now redundant